environments/india/inventory.ini

```diff
@@ -12,6 +12,39 @@ datavol_device='/dev/mapper/consolidated-data'
 [django1]
 10.162.36.231 hostname="django1"
 
+[citusdb0]
+10.162.36.194
+
+[citusdb0:vars]
+hostname="citusdb0"
+
+[citusdb1]
+10.162.36.212
+
+[citusdb1:vars]
+hostname="citusdb1"
+
+[citusdb2]
+10.162.36.195
+
+[citusdb2:vars]
+hostname="citusdb2"
+
+[citusdb_worker:children]
+citusdb1
+citusdb2
+
+[citusdb_worker:vars]
+ansible_python_interpreter = '/usr/bin/python3'
+datavol_device=/dev/xvdc
+
+[citusdb_cordinator:children]
+citusdb0
+
+[citusdb_cordinator:vars]
+ansible_python_interpreter = '/usr/bin/python3'
+datavol_device=/dev/xvdc
```

Comment:
these changes are unrelated to removing the pillow

Reply:
checkout from wrong branch.